content/docs/setup/kubernetes/helm-install.md

```diff
@@ -23,6 +23,10 @@ recommended.
 
 1. [Install the Helm client](https://docs.helm.sh/using_helm/#installing-helm).
 
+1. Istio by default uses LoadBalancer service object types.  Some platforms do not support LoadBalancer
+   service objects.  For platforms lacking LoadBalancer support, install Istio with NodePort support
+   instead with the flags `--set ingress.service.type=NodePort ingressgateway.service.type=NodePort egressgateway.service.type=NodePort` appended to the end of the helm operation.
```

Comment:
I had to insert commas:

`--set ingress.service.type=NodePort,ingressgateway.service.type=NodePort,egressgateway.service.type=NodePort`

 Otherwise I would get:

`"argument: chart name" required `

From helm's documentation:

https://github.com/kubernetes/helm/blob/master/docs/using_helm.md

`Multiple values are separated by , characters. So --set a=b,c=d ...`

```
$ helm version
Client: &version.Version{SemVer:"v2.7.0", GitCommit:"08c1144f5eb3e3b636d9775617287cc26e53dba4", GitTreeState:"clean"}

```

Also, I'm wondering if it would be an idea for there to be an automated test which validates the documentation?